### PR TITLE
Add support to run dazel from windows enviroment

### PR DESCRIPTION
**Pull request by luisabv**
Changed to use python libraries instead of unix commands.
Changed docker commands to make the queries more reliable
Changed handling of paths using pathlib to make dazel platform independent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Running the command for the first time will start the container on it's own, and
 You can configure anything you need through the ".dazelrc" file in the same directory.
 Take a look at the configuration section for information on how to write one.
 
+It is even possible to run dazel from Windows enviroment towards a Posix Docker image. You just need to run dazel from a terminal started as Administrator to be able to create links.
```
Comment (gustafj) [inline README.md]: environment

Comment (gustafj) [reply README.md]: I would switch from "even" to "also"

**Comment by persmy** [comment]
From what I understand it looks good to me.

**Review by gustafj — COMMENTED** [review]
Two extremely minor things, looks awesome @luisabv !
Great work! :+1:

File: README.md (hunk 1 above)
Comment (luisabv) [reply README.md]: Fixed

**Review by gustafj — APPROVED** [review]
:+1:

**Pull request merged**